src/tests/Myr_random.test.js

```diff
@@ -0,0 +1,123 @@
+import Myr from "../myr/Myr";
+
+const myr = new Myr();
+
+describe("Random Function Tests", () => {
+    
+    //randomInt

```

Comment:
Can we check that we actually only get integers?

Reply:
Added check